docs/faq.md

```diff
@@ -192,3 +192,12 @@ In the EarlGrey test target, import the application's app delegate header. In `s
     [delegate resetApplicationForTesting];
 }
 ```
+
+**How do I match on internal classes?**
+
+Use `NSClassFromString` to match on internal classes (prefixed with `_`) that
```

Comment:
There are UIKit classes that are not prefixed with _ but still considered internal / private because they're not directly accessible.